app/Website/src/Mail/ContactMail.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Website\Mail;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class ContactMail extends Mailable
+{
+    use Queueable, SerializesModels;
+    public array $contactData;
+    /**
+     * Create a new message instance.
+     */
+    public function __construct(array $contactData)
```

Comment:
Please use constructor property promotion: https://stitcher.io/blog/constructor-promotion-in-php-8

Basically just: `public function __construct(public array $contactData) {}` and remove line 11 and 17